### ISSUE TITLE
Extensions of ConcordionRunner are not detected by JUnit4FrameworkProvider

### ISSUE THREAD

**Issue by gdubya** [concordion/concordion#251]
We use the [SpringifiedConcordionRunner](https://github.com/chiknrice/concordion-spring-runner) and the FixtureSpecificationMapper now fails in 2.1.0 because org.concordion.integration.junit4.JUnit4FrameworkProvider#isConcordionFixture returns false for these fixtures. Wouldn't it be better to rewrite the provider to account for inheritance (isAssignableFrom)? e.g.

```
public class JUnit4FrameworkProvider implements TestFrameworkProvider {
    @Override
    public boolean isConcordionFixture(Class<?> clazz) {
        RunWith annotation = clazz.getAnnotation(RunWith.class);
        return annotation != null && ConcordionRunner.class.isAssignableFrom(annotation.value());
    }
```

**Comment by nigelcharman** [comment]
Hi Gareth, yes I totally agree. In fact, PR #250 from @dadrus includes
this fix. This will likely be pulled into the code in the next few days
and a new release made.

**Comment by gdubya** [comment]
Great! Thanks Nigel! I'll close this issue then since it's already covered in the PR.

**Comment by nigelcharman** [comment]
Released in Concordion 2.1.1
